google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java

```diff
@@ -188,8 +194,19 @@ private void showRightClickMenu(MouseEvent event) {
     Blob selectedBlob = tableModel.getBlobAt(bucketContentTable.rowAtPoint(event.getPoint()));
 
     if (selectedBlob != null) {
+      copyBucketNameMenuItem.addActionListener(
+          e ->
+              UsageTrackerProvider.getInstance()
+                  .trackEvent(GctTracking.GCS_BUCKET_LIST_COPY_BUCKET_NAME)
+                  .ping());
+
+      copyBlobNameMenuItem.addActionListener(
+          e -> pingMetrics(GctTracking.GCS_BLOB_BROWSE_COPY_BLOB_NAME));
       copyBlobNameMenuItem.addActionListener(
           new CopyToClipboardActionListener(selectedBlob.getName()));
+
+      copyBucketNameMenuItem.addActionListener(
```

Comment:
It looks like this won't be differentiated from the `GCS_BUCKET_LIST_COPY_BUCKET_NAME` metric because the `copyBucketNameMenuItem` is the same instance, right?